src/web100srv.c

```diff
@@ -525,43 +513,35 @@ void cleanup(int signo) {
       }
       exit(0);
     case SIGPIPE:
+      // SIGPIPE is an expected signal due to race conditions regarding the
+      // possibility of writing a message to an already-terminated child.  Do
+      // not let it kill the process.  The SIGCHLD handler will take care of
+      // child process cleanup.
       fp = fopen(get_logfile(), "a");
       if ((fp != NULL) && (get_debuglvl() > 4)) {
-        fprintf(
-            fp,
-            "Received SIGPIPE signal: terminating active web100srv process "
-            "[%d]\n", getpid());
+        fprintf(fp, "Received SIGPIPE: a child has terminated early.\n");
         fclose(fp);
       }
-      sig13 = 1;
       break;
 
     case SIGHUP:
-      /* Initialize Web100 structures */
+      // Initialize Web100 structures
       count_vars = tcp_stat_init(VarFileName);
 
-      /* The administrator view automatically generates a usage page for the
-       * NDT server.  This page is then accessable to the general public.
-       * At this point read the existing log file and generate the necessary
-       * data.  This data is then updated at the end of each test.
-       * RAC 3/11/04
-       */
-      if (admin_view == 1)
-        view_init(refresh);
+      // The administrator view automatically generates a usage page for the
+      // NDT server.  This page is then accessable to the general public.
+      // At this point read the existing log file and generate the necessary
+      // data.  This data is then updated at the end of each test.
+      if (admin_view == 1) view_init(refresh);
       break;
 
     case SIGCHLD:
-      /* moved actions to child_sig() routine on 3/10/05  RAC
-       * Now all we do here is set a flag and return the flag
-       * is checked near the top of the main wait loop, so it
-       * will only be accessed once and only the testing proces
-       * will attempt to do something with it.
-       */
-      /* sig17 = 1; */
+      // Set a flag. The flag is checked near the top of the main wait loop, so
```

Comment:
Looks like we're double spacing after the period in this file, so double space for consistency even though it's silly.